downstream/modules/platform/ref-operator-crs.adoc

```diff
@@ -649,3 +649,19 @@ spec:
 # * {LightspeedShortName} is not deployed
 ----
 
+== EDA_MAX_RUNNING_ACTIVATIONS
+
+[subs="+attributes"]
+----
+---
+apiVersion: aap.ansible.com/v1alpha1
+kind: AnsibleAutomationPlatform
+metadata:
+  name: myaap
+spec:
+  eda:
+    extra_settings:
+      - setting: EDA_MAX_RUNNING_ACTIVATIONS
+        value: "12"
```

Comment:
```suggestion
        value: "15"
```
Recommend we give a number here as an example, that is different from the default.